flowzone.yml

```diff
@@ -1367,6 +1367,7 @@ jobs:
           github.event.pull_request.merged == true
           && steps.create_tag.outputs.sha != ''
         env:
+          GH_TOKEN: ${{ steps.gh_app_token.outputs.token || secrets.FLOWZONE_TOKEN }}
```

Comment:
[This](https://github.com/balena-io/environment-production/blob/fb45587920dd21973295af67b1ba523f525fd1d3/.github/workflows/flowzone.yml#L11-L24) is why

Reply:
yep, I get it now, the default token permissions are used otherwise, which in that workflow are all `read`

Reply:
So we can close this PR right, and just adjust the permissions of the automatic GITHUB_TOKEN on that repo?

I would prefer we use the automatic token where it makes sense to do so, and only use the Flowzone token for cases where special permissions are required (like cross-repo permissions)